scripts/migrate_score_override.py

```diff
@@ -0,0 +1,138 @@
+import itertools
+from django.contrib.auth.models import User
+from django.utils.timezone import now
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from openassessment.assessment.models import Assessment, AssessmentPart, StaffWorkflow
+from openassessment.workflow.models import AssessmentWorkflow, AssessmentWorkflowStep
+from student.models import anonymous_id_for_user
+from submissions.models import Score, ScoreAnnotation, Submission
+
+old_scores = Score.objects.filter(submission__isnull=True, reset=False)
```

Comment:
Scripts should generally not contain the logic directly in the body, but rather, should take a form like:

```python
def main():
    # do stuff here

if __name__ == '__main__':
    main()
```

The idea is that if another file were to import this file,
as is, the data migration would be run automatically.
Whereas, we only want to run this if the script is executed directly,
not if it's just imported.

I don't know how/if this is affected by the `execfile` call,
but we should consider using this idiom.